src/mxarray.jl

```diff
@@ -406,8 +406,13 @@ end
 # char arrays and string
 
 function mxarray(s::String)
-    pm = ccall(_mx_create_string, Ptr{Void}, (Ptr{UInt8},), s)
-    MxArray(pm)
+    utf16string = transcode(UInt16, s)
+    pm = ccall(_mx_create_char_array, Ptr{Void}, (mwSize, Ptr{mwSize},), 2,
+               _dims_to_mwSize((1, length(utf16string))))
+    mx = MxArray(pm)
+    ccall(:memcpy, Ptr{Void}, (Ptr{Void}, Ptr{Void}, UInt), data_ptr(mx), utf16string,
```

Comment:
The convention is usually ["try to adhere to a 92 character line length limit"](https://github.com/JuliaLang/julia/blob/master/CONTRIBUTING.md#general-formatting-guidelines-for-julia-code-contributions)